libs/components/src/lib/toggletip/toggletip.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { ADD_TEMPLATE_TO_FIXTURE, elementUpdated, fixture } from '@vivid-nx/shared';
+import { FoundationElementRegistry } from '@microsoft/fast-foundation';
+import type { Button } from '../button/button';
+import { Toggletip } from './toggletip';
+import '../button';
+import '.';
+import { toggletipDefinition } from './definition';
+
+const COMPONENT_TAG = 'vwc-toggletip';
+
+describe('vwc-toggletip', () => {
+	let element: Toggletip;
+	let anchor: Button;
+
+	global.ResizeObserver = jest.fn()
+		.mockImplementation(() => ({
+			observe: jest.fn(),
+			unobserve: jest.fn(),
+			disconnect: jest.fn()
+		}));
+
+	beforeEach(async () => {
+		element = fixture(
+			`<${COMPONENT_TAG}></${COMPONENT_TAG}>`
+		) as Toggletip;
+
+		anchor = fixture(
+			'<vwc-button id="anchorButton"></vwc-button>', ADD_TEMPLATE_TO_FIXTURE
+		) as Button;
+	});
+
+	describe('basic', () => {
+		it('should be initialized as a vwc-toggletip', async () => {
+			expect(toggletipDefinition()).toBeInstanceOf(FoundationElementRegistry);
+			expect(element).toBeInstanceOf(Toggletip);
+
+			expect(element.open).toBe(false);
+			expect(element.alternate).toBe(false);
+			expect(element.headline).toBeUndefined();
+			expect(element.placement).toBe('right');
+		});
+	});
+
+	describe('open/close', () => {
```

Comment:
Describe the API. There's no `close` property or method.
```suggestion
	describe('open', () => {
```